src/OpenLoco/src/Ui/Windows/ObjectSelectionWindow.cpp

```diff
@@ -276,7 +277,6 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
         Widgets::ImageButton({ 585, 2 }, { 13, 13 }, WindowColour::primary, ImageIds::close_button, StringIds::tooltip_close_window),
         Widgets::Panel({ 0, 42 }, { 600, 356 }, WindowColour::secondary),
 
-        // Primary tab area
```

Comment:
Presumably removed by mistake in rebase?

Reply:
Ops.